src/components/storagePools/storagePoolDelete.jsx

```diff
@@ -160,26 +162,32 @@ export class StoragePoolDelete extends React.Component {
 
         const defaultBody = (
             <Stack hasGutter>
-                <Form isHorizontal>
-                    { storagePool.active && volumes.length > 0 &&
-                    <FormGroup label={_("Delete content")} fieldId='storage-pool-delete-volumes' hasNoPaddingTop>
-                        <Checkbox id='storage-pool-delete-volumes'
-                                  isChecked={this.state.deleteVolumes}
-                                  label={_("Delete the volumes inside this pool")}
-                                  onChange={checked => this.onValueChanged('deleteVolumes', checked)} />
-                    </FormGroup>}
-                    { !storagePool.active && _("Deleting an inactive storage pool will only undefine the pool. Its content will not be deleted.")}
-                </Form>
-                { storagePool.active && showWarning() }
+                { storagePool.active
+                    ? (volumes.length > 0
+                        ? <Checkbox id='storage-pool-delete-volumes'
+                                    isChecked={this.state.deleteVolumes}
+                                    label={<>
+                                        { _("Also delete all volumes inside this pool: ")}
```

Comment:
Trailing space. This is important for i18n'ed strings, as it's basically guaranteed that this won't survive translations. If you need the space for layouting, add it *after* the gettext call.

Reply:
Ah this was probably typo, as list is in a new line, there is no need for a space